website/docs/en/guide/basic/static-assets.mdx

```diff
@@ -112,10 +112,10 @@ For example, you can place files such as `robots.txt`, `manifest.json`, or `favi
 
 You can reference files in the `public` directory via a URL.
 
-For example, in an HTML template, the `./public/favicon.ico` file can be referenced as `/favicon.ico`.
+For example, in an HTML template, the `./public/favicon.ico` file can be referenced as `/favicon.ico`, [BASE_URL](/guide/advanced/env-vars#processenvpublic_base_path) is the base path of the server.
 
 ```html title="index.html"
-<link rel="icon" href="/favicon.ico" />
+<link rel="icon" href="<%= process.env.BASE_URL %>/favicon.ico" />
```

Comment:
BASE_URL + '/' = double slash?

Reply:
Will automatically remove the trailing slash from `BASE_URL`.

I prefer to consistent with [process.env.ASSET_PREFIX](https://rsbuild.dev/guide/advanced/env-vars#processenvasset_prefix).

Reply:
I tend to align with the ecosystem, we can check Next.js / Vite before making a decision.

Reply:
Both vite and next.js are output to the front-end env as is according to the `baseUrl` configured by the user.

So, I think we don't need to do anything to `process.env.BASE_URL`, just leave it as is.

Reply:
LGTM